### PR TITLE
fix in reporting, mkdirs, docs + example

### PR DESCRIPTION
**Pull request by tjungblu**

### CODE REVIEW

File: pkg/cli/cli.go [hunk 1]
```diff
@@ -97,15 +100,16 @@ func Run(configPath string, inputPath string, outputPath string, deleteOutputFol
 	}
 
 	report := walker.GenerateReport()
-	rPath := filepath.Join(outputPath, reportFileName)
-	reportFile, err := os.Create(rPath)
+	// we store the report in the working dir of the CLI, it should never be created in the output folder
```
Comment (arjunrn) [inline pkg/cli/cli.go:103]: This comment describes the change being made is not actually useful when read by itself.

Comment (arjunrn) [reply pkg/cli/cli.go:103]: Also if you want to store it in another location then use a flag which the user can set.

Comment (tjungblu) [reply pkg/cli/cli.go:103]: it was more intended to be a warning for others changing. But it totally makes sense to add a flag for that, will do later.

Comment (tjungblu) [reply pkg/cli/cli.go]: done

**Pull request merged**